root/app/snippies/ad.py

```diff
@@ -1,13 +1,14 @@
 from ldap3 import Server, Connection, ALL
-from snippies import db, app_log, config
+from snippies import db, app_log
 import os
 
 # AD Configuration
 AD_SERVER = 'WDC01.woodleighschool.net'
-AD_USERNAME = config.get_credential("ADUSERNAME")
```

Comment:
loading variables from a `ini` is redundant with env vars defined in the docker compose, with the same security implicartions, we can also map [secrets](https://github.com/linuxserver/docker-code-server?tab=readme-ov-file#environment-variables-from-files-docker-secrets), which is the most 'secure' method